docs/user-guide/kubectl/kubectl_get.md

```diff
@@ -80,6 +80,7 @@ $ kubectl get rc/web service/frontend pods/web-pod-13je7
 ### Options
 
 ```
+  -a, --all=false: Show all pods (default shows just pending and running ones.)
```

Comment:
Is this valid only for pods?
The description should be generic for any resource